### ISSUE TITLE
issue in ubuntu 20

### ISSUE THREAD

**Issue by johnfelipe** [finloop/airflow-postgres-superset-on-docker#8]
```
root@ubuntu20dockers:~/airflow-postgres-superset-on-docker# docker-compose up
ERROR: The Compose file './docker-compose.yaml' is invalid because:
Unsupported config option for services.airflow-cli: 'profiles'
Invalid top-level property "x-airflow-common". Valid top-level sections for this Compose file are: version, services, networks, volumes, and extensions starting with "x-".

You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
services.airflow-scheduler.depends_on contains an invalid type, it should be an array
services.airflow-triggerer.depends_on contains an invalid type, it should be an array
services.airflow-webserver.depends_on contains an invalid type, it should be an array
services.airflow-worker.depends_on contains an invalid type, it should be an array
services.flower.depends_on contains an invalid type, it should be an array
services.airflow-cli.depends_on contains an invalid type, it should be an array
services.airflow-init.depends_on contains an invalid type, it should be an array
root@ubuntu20dockers:~/airflow-postgres-superset-on-docker#

```

**Comment by finloop** [comment]
Hi, I've encountered this issue in another repository. Try replacing this ile in docker-compose.yaml:
```docker
version: '3'
```
with
```docker
version: '3.3'

**Comment by johnfelipe** [comment]
```
root@ubuntu20dockers:~/airflow-postgres-superset-on-docker# docker-compose up
ERROR: The Compose file './docker-compose.yaml' is invalid because:
Unsupported config option for services.airflow-cli: 'profiles'
Invalid top-level property "x-superset-volumes". Valid top-level sections for this Compose file are: version, services, networks, volumes, secrets, configs, and extensions starting with "x-".

You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
services.airflow-scheduler.depends_on contains an invalid type, it should be an array
services.airflow-triggerer.depends_on contains an invalid type, it should be an array
services.airflow-webserver.depends_on contains an invalid type, it should be an array
services.airflow-worker.depends_on contains an invalid type, it should be an array
services.flower.depends_on contains an invalid type, it should be an array
services.airflow-cli.depends_on contains an invalid type, it should be an array
services.airflow-init.depends_on contains an invalid type, it should be an array
root@ubuntu20dockers:~/airflow-postgres-superset-on-docker#

```

same issue after change version

**Comment by finloop** [comment]
What version of docker-compose you're using?
```bash
docker-compose --version
```
This repository works with versions >=1.27.0 ([CHANGELOG](https://github.com/docker/compose/releases/tag/1.27.0))

**Comment by johnfelipe** [comment]
`docker-compose version 1.25.0, build unknown`

how can update?

**Comment by finloop** [comment]
Here's a list of articles:
-  https://phoenixnap.com/kb/install-docker-compose-on-ubuntu-20-04
- https://stackoverflow.com/questions/49839028/how-to-upgrade-docker-compose-to-latest-version

TL;DR It depends on how you've installed it. I'm closing the issue for now.
